pype/settings/entities/schemas/projects_schema/schema_project_nuke.json

```diff
@@ -6,6 +6,32 @@
     "is_file": true,
     "children": [
         {
+            "type": "dict",
+            "collapsible": true,
+            "key": "triggers",
```

Comment:
Let's make this a bit less particular so we can keep adding settings without coming up with new names every time. 
I reckon the same as in Harmony, Where we have a `general` category inside the hosts, for anything that doesn't relate to plugins.

I'll adjust maya to reflect that too. 

```
"nuke":{
  "general": {
      "setting1" : "",
      "setting2: "",
      "create_initial_workfile": true,
}
}
```

Reply:
Does this mean I should move menu shortcuts to the general settings too?

Reply:
Unfortunatelly they will have to be per host, Simply to avoid potential conflicts with native hosts hotkeys